org-chart/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,69 @@
+import { useContext } from "react";
+import './Card.css';
+import { AvatarContext } from "../../contexts/AvatarContext";
+import { IoPeople } from 'react-icons/io5';
+
+type Border = {
+    borderWidth: string,
+    borderColor: string,    
+}
+
+type Member =  {
+    name: string, 
+    skill: string,
+    nextAvatarTeam?: string,
+    imgSrc: string,
+    id: string
+}
+
+type CardProps = {
+    borderDetails: Border,
+    onClick: () => void,
+    member: Member
+}
+
+function Card({
+    borderDetails, 
+    onClick,
+    member
+}:CardProps)
+{
+    const context = useContext(AvatarContext)
+    const seeMore = member?.nextAvatarTeam ? <IoPeople/> : ""
+
+    const connectionLinesBottomPart = member.id != "1" ? (           
```

Comment:
```suggestion
    const connectionLinesBottomPart = member.id !== "1" ? (           
```

This seems like a code smell to me, `"1"` can probably just be a numeric ID of type `number`?